Add some translate i18n + French translate

gilardh opened the pull request.

gilardh committed:
Add some translate i18n + French translate

gilardh committed:
Update of 'Transaction.vue' page

- Update of financial calculations
- Enlarging input fields

leto commented on src/renderer/components/Wallet/Transactions.vue:
```diff
@@ -283,18 +293,39 @@
-        var shieldedXtn = 0;
+      updateTransactionForm (form,availableBalance,input_from) {
+
+        //console.log('from=' + input_from);
```
Please remove this leftover debug stuff

gilardh committed:
Improved UI of the page 'Transactions.vue'

Restriction of the fields 'amount', 'dev fee' and 'fee transaction':
- Entry only numeric type
- Entry of a single dot
- Maximum number of decimal at 8

Round the dev fee to 8 decimal

gilardh committed:
(cancels & replaces) Restriction of the fields 'amount', 'dev fee' and
'fee transaction':
- Entry only numeric type
- Entry of a single dot
- Maximum number of decimal at 8

Round the dev fee to 8 decimal

gilardh committed:
Uniformity of the user interface of the settings page

- Identical top banner and wallpaper on the 'Wallet' and 'Contacts' pages
- Addition of the closing button of the application at the top left

leto commented:
@gilardh thanks for all your work!
